t/plugin/kafka-logger.t

```diff
@@ -1208,3 +1208,107 @@ hello world
 --- error_log_like eval
 qr/send data to kafka: \{.*"body":"abcdef"/
 --- wait: 2
+
+
+
+=== TEST 28: hit route, not trigger request_body_expr rule
+--- request
+GET /hello
```

Comment:
The test will pass whether the request_body_expr is correct or not

Reply:
Ok, i just want to prove that if there was no request body, it still work normally. There is already has a eval false case below and i'm delete this case for good